i18n/ja/docusaurus-plugin-content-docs/current/getting-started.md

```diff
@@ -32,18 +33,20 @@ brew install momento-cli
 
 **Linux**
 
-1. Download the latest linux tar.gz file from [https://github.com/momentohq/momento-cli/releases/latest](https://github.com/momentohq/momento-cli/releases/latest)
-2. Unzip the file: `tar -xvf momento-cli-X.X.X.linux_x86_64.tar.gz`
-3. Move `./momento` to your execution path.
+1. 最新の Linux 用 tar.gz ファイルをこちらからダウンロード: [https://github.com/momentohq/momento-cli/releases/latest](https://github.com/momentohq/momento-cli/releases/latest)
+2. そのファイルを解凍: `tar -xvf momento-cli-X.X.X.linux_x86_64.tar.gz`
```

Comment:
It doesn't make sense. `tar -xvf momento-cli-X.X.X.linux_x86_64.tar.gz` is a shell command that the customers can copy and paste (actually, it won't work verbatim because of the version string anyway).

If it's confusing, I would say:

> 次のコマンドでそのファイルを解凍: `tar -xvf momento-cli-X.X.X.linux_x86_64.tar.gz`